ompi/mpi/c/abi_converters.h

```diff
@@ -561,6 +561,34 @@ __opal_attribute_always_inline__ static inline void ompi_convert_intern_status_a
     memcpy(ptr, &inp->_ucount,sizeof(inp->_ucount));
 }
 
+__opal_attribute_always_inline__ static inline int ompi_convert_abi_ts_level_intern_ts_level(int ts_level)
+{
+    if (MPI_THREAD_SINGLE_ABI_INTERNAL == ts_level) {
```

Comment:
Wouldn't a `switch` statement be better? It may ultimately be a matter of taste (optimizing compilers may generate similar binary code). Just a mild observation.